app/src/main/java/eu/kanade/tachiyomi/data/backup/models/AbstractBackupManager.kt

```diff
@@ -0,0 +1,7 @@
+package eu.kanade.tachiyomi.data.backup.models
```

Comment:
Probably shouldn't be in `models`

Reply:
Where should the base models for the backup system go?